features/video.yml.dist

```diff
@@ -0,0 +1,339 @@
+  # support:
+  #   chrome: "3"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "20"
+  #   firefox_android: "20"
+  #   safari: "3.1"
+  #   safari_ios: "3"
+  - api.HTMLMediaElement.defaultPlaybackRate
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "5"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "4"
+  #   firefox_android: "4"
+  #   safari: "5"
+  #   safari_ios: "5"
+  - api.HTMLMediaElement.preload
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "15"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "11"
+  #   firefox_android: "14"
+  #   safari: "6"
+  #   safari_ios: "6"
+  - api.HTMLMediaElement.defaultMuted
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "30"
+  #   chrome_android: "30"
+  #   edge: "12"
+  #   firefox: "11"
+  #   firefox_android: "14"
+  #   safari: "5"
+  #   safari_ios: "4.2"
+  - html.elements.video.muted
+
+  # baseline: high
+  # baseline_low_date: 2016-09-13
+  # baseline_high_date: 2019-03-13
+  # support:
+  #   chrome: "3"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "3.5"
+  #   firefox_android: "4"
+  #   safari: "3.1"
+  #   safari_ios: "10"
+  - html.elements.video.autoplay
+
+  # baseline: high
+  # baseline_low_date: 2016-09-20
+  # baseline_high_date: 2019-03-20
+  # support:
+  #   chrome: "33"
+  #   chrome_android: "33"
+  #   edge: "13"
+  #   firefox: "22"
+  #   firefox_android: "22"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.HTMLMediaElement.crossOrigin
+
+  # baseline: high
+  # baseline_low_date: 2018-04-30
+  # baseline_high_date: 2020-10-30
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "17"
+  #   firefox: "53"
+  #   firefox_android: "53"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - api.HTMLMediaElement.play.returns_promise
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "3"
+  #   chrome_android: "18"
+  #   edge: "79"
+  #   firefox: "3.5"
+  #   firefox_android: "4"
+  #   safari: "3.1"
+  #   safari_ios: "3"
+  - api.HTMLVideoElement.resize_event
+
+  # baseline: high
+  # baseline_low_date: 2020-07-28
+  # baseline_high_date: 2023-01-28
+  # support:
+  #   chrome: "33"
+  #   chrome_android: "33"
+  #   edge: "18"
+  #   firefox: "74"
+  #   firefox_android: "79"
+  #   safari: "10"
+  #   safari_ios: "10"
+  - html.elements.video.crossorigin
+
+  # baseline: false
+  # support:
+  #   chrome: "3"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "3.5"
+  #   firefox_android: "4"
+  #   safari: "3.1"
+  - api.HTMLMediaElement.volume
+
+  # baseline: false
+  # support:
+  #   safari: "3.1"
+  #   safari_ios: "3"
+  - api.HTMLMediaElement.playbackRate
```

Comment:
FWIW, both of these have dodgy-looking data. Someday someone might come along and fix it 😄

Reply:
`playbackRate` is about only Safari supporting a negative playback rate. The spec does allow for throwing an exception "if the given value is not supported by the user agent" but doesn't require any specific playback rates to be supported.

`volume` is about it not doing anything on iOS.